easybuild/easyconfigs/s/SQANTI3/SQANTI3-0.1-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,93 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Research Center - Seattle - US
+#
+easyblock = 'PythonBundle'
+
+name = 'SQANTI3'
+version = '0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ConesaLab/SQANTI3'
+description = """SQANTI3 is the first module of the Functional IsoTranscriptomics (FIT) framework,
+ that also includes IsoAnnot and tappAS. Used for new long read-defined transcriptome."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('scikit-learn', '0.21.3', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('PyYAML', '5.1.2'),
+    ('JupyterLab', '1.2.5', versionsuffix),
+    ('Pysam', '0.15.3'),
+    ('BEDTools', '2.29.2'),
+    ('SQLite', '3.29.0'),
+    ('tbb', '2019_U9'),
+    ('Tk', '8.6.9'),
+    ('unixODBC', '2.3.7'),
+    ('XZ', '5.2.4'),
+    ('ZeroMQ', '4.3.2'),
+    ('zlib', '1.2.11'),
+    ('zstd', '1.4.4'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('click', '7.1.2', {
+        'checksums': ['d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a'],
+    }),
+    ('semantic-version', '2.8.5', {
+        'modulename': 'semantic_version',
+        'source_tmpl': 'semantic_version-%(version)s.tar.gz',
+        'checksums': ['d2cb2de0558762934679b9a104e82eca7af448c9f4974d1f3eeccff651df8a54'],
+    }),
+    ('pyPEG2', '2.15.2', {
+        'checksums': ['2b2d4f80d8e1a9370b2a91f4a25f4abf7f69b85c8da84cd23ec36451958a1f6d'],
+    }),
+    ('cmakelists_parsing', '0.3.1', {
+        'checksums': ['cd48d5d42eeab07e58af8c7219a6f0096e58d5fa8975524ca47c14967ae223ad'],
+    }),
+    ('cached-property', '1.5.1', {
+        'checksums': ['9217a59f14a5682da7c4b8829deadbfc194ac22e9908ccf7c8820234e80a1504'],
+    }),
+    ('conan', '1.28.1', {
+        'modulename': False,
+        'checksums': ['9024ad0acb6f046ee00e91392a8dbe31215a322975e212e2e9fc9337e6487d28'],
+    }),
+    ('toolz', '0.10.0', {
+        'checksums': ['08fdd5ef7c96480ad11c12d472de21acd32359996f69a5259299b540feba4560'],
+    }),
+    ('pydantic', '1.6.1', {
+        'checksums': ['54122a8ed6b75fe1dd80797f8251ad2063ea348a03b77218d73ea9fe19bd4e73'],
+    }),
+    ('cupcake', '0.0.4', {
+        'use_pip': False,
+        'checksums': ['84f87a95f21ccf4af59805bab3b01ae0b2f1a06c1014a5d55f25af00df7e8d35'],
+    }),
+    ('sortedcontainers', '2.2.2', {
+        'checksums': ['4e73a757831fc3ca4de2859c422564239a31d8213d09a2a666e375807034d2ba'],
+    }),
+    ('sortedcollections', '1.2.1', {
+        'checksums': ['58c31f35e3d052ada6a1fbfc235a408e9ec5e2cfc64a02731cf97cac4afd306a'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
```

Comment:
The tests fail because these `sanity_check_paths` are the default ones and can be removed
```suggestion
```

Reply:
snaity_check_paths removed